docs/UserGuide.md

```diff
@@ -106,19 +77,48 @@ Parameter:
 * `c/CHARACTERISTIC` (optional)  : The characteristics of the property the customer is looking for (String)
 
 Examples:
-* `addcust n/Fredy p/12345678 e/fredylawrence@gmail.com b/100000`
+* `addcust n/Fredy p/12345678 e/fredylawrence@gmail.com b/1000000`
 * `addcust n/Boedi p/88888888 e/boedi@gmail.com b/250000 c/white`
 * `addcust n/Phoebe p/87654321 e/pb@gmail.com b/200000`
 
 When command succeeds:
-* `Customer NAME has been added`
+* `New customer added:Name; Phone:PHONE; Email:EMAIL; Budget:BUDGET; Tags:[TAGS]…​`
 
 When command fails:
 * `Missing name parameter for add customers command` for missing name parameter
 * `Missing phone parameter for add customers command` for missing phone parameter
 * `Missing email parameter for add customers command` for missing email parameter
 * `Invalid Command` for misspelling of command
 
+### Adding a property: `addprop`
+
+Adds a property to the application.
+
+Format: `addprop n/NAME a/ADDRESS [c/CHARACTERISTIC] ph/number pr/budget`
+
+Parameter:
+* `n/NAME`				         : The propName of the property (String)
```

Comment:
I think `propName` can be changed to `name` instead, users probably wouldn't understand `propName`